data/attributes.yml

```diff
@@ -5042,24 +5042,24 @@ base_attributes:
   - nut_butter_variety__pistachio
   - nut_butter_variety__walnut
 - id: 1492
-  name: Dry bean variety
-  description: Identifies the variety of dry beans used, e.g. chickpeas, black beans
-  friendly_id: dry_bean_variety
```

Comment:
We definitely can't change this handle without a mapping. As merchants will have metafields already based on this.